src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.data.TaskList;
+import duke.data.task.Task;
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+public class MarkCommand extends ModifyCommand {
+    public static final String COMMAND_WORD = "MARK";
+    private static final boolean IS_MARK_METHOD = true;
```

Comment:
Could this be renamed to be more descriptive?

Reply:
The IS_MARKED_METHOD is now removed after refactoring the code for display message (see reply below)